frontends/p4/simplify.cpp

```diff
@@ -61,6 +61,16 @@ const IR::Node* DoSimplifyControlFlow::postorder(IR::BlockStatement* statement)
 
 const IR::Node* DoSimplifyControlFlow::postorder(IR::IfStatement* statement)  {
     LOG3("Visiting " << dbp(getOriginal()));
+    if (auto lnot = statement->condition->to<IR::LNot>()) {
+        // swap branches
+        statement->condition = lnot->expr;
+        auto e = statement->ifFalse;
+        if (!e)
+            e = new IR::EmptyStatement();
```

Comment:
Shouldn't if statements have brackets?

Reply:
No, our style guide does not require brakets for a single statement.
cpp lints decides here.